package.json

```diff
@@ -1119,7 +1119,8 @@
     "lint": "npm run lint:src && npm run lint:tests  && npm run markdown",
     "lint:src": "tslint --format stylish --exclude \"**/__tests__/**/*.ts\" \"src/**/*.ts\"",
     "lint:tests": "tslint --format stylish \"**/__tests__/**/*.ts\"",
-    "clean": "gulp clean"
+    "clean": "gulp clean",
+    "d2u": "node ./scripts/d2uChangelog.js"
```

Comment:
If you plan to add `d2u` as a script, then we do need to include the ssp-dos2unix as a `devDep`.
However, I believe you can just call `node ./scripts/d2uChangelog.js` instead of `npm run d2u` on the Jenkinsfile?

Reply:
Changed to call node directly from Jenkinsfile